imagetest/test_suites/artifact_registry/artifact_registry_test.go

```diff
@@ -0,0 +1,109 @@
+package artifactregistry
+
+import (
+	"flag"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	aptCreatePrivateRepo = `cat | sudo tee /etc/apt/sources.list.d/artifact-registry-private-repo.list <<EOF
+deb [ trusted=yes ] ar+https://us-central1-apt.pkg.dev/projects/bct-prod-images apt main
+EOF
+sudo DEBIAN_FRONTEND=noninteractive apt update
+`
+
+	yumCreatePrivateRepo = `cat | sudo tee /etc/yum.repos.d/artifact-registry-private-repo.repo <<EOF
```

Comment:
as we discussed, there's no reason to encapsulate this logic with shell commands. we can just write the appropriate file  contents directly.